hs_core/hydroshare/resource.py

```diff
@@ -425,6 +433,16 @@ def create_resource(
         if not metadata:
             metadata = []
 
+        fed_zone_home_path = ''
+        if fed_res_path:
```

Comment:
Is this the user case where user is trying to create a resource on a federated zone, without any content files for that resource?